.travis.yml

```diff
@@ -1,7 +1,7 @@
 dist: bionic
 os: linux
 language: python
-python: 3.6
+python: 3.8
```

Comment:
Is there a reason we decided to use 3.8 versus the later verssions? Per discussion earlier, I am using 3.9 in the kmr2-5.2.7 branch.

Reply:
@snaiksat no specific reason as we can use any python version >= 3.8. Will use python 3.9